packages/smooth_app/lib/pages/product/new_product_page.dart

```diff
@@ -219,33 +226,133 @@ class _ProductPageState extends State<NewProductPage> {
           top: 4.0,
           bottom: 20.0,
         ),
-        insets: const EdgeInsets.all(12.0),
-        child: Column(children: <Widget>[
-          Align(
+        insets: EdgeInsets.zero, // Zero padding for the card content.
+        clipBehavior: Clip.antiAliasWithSaveLayer,
+        child: Column(
+          children: <Widget>[
+            getProductMatchHeader(context),
+            Container(
+              padding: const EdgeInsets.symmetric(horizontal: 12.0),
+              child: Column(children: [
+                getProductTitleTile(context),
+                for (final Attribute attribute in scoreAttributes)
+                  ScoreAttributeCard(
+                      attribute: attribute, iconHeight: iconHeight),
```

Comment:
I'll mad the function call fit in a single line instead, so many lines inside the for loop makes it a bit harder to parse (specially since we don't have braces)